src/shotgunEventDaemon.py

```diff
@@ -961,6 +985,14 @@ def process(self, event):
             if self._stopOnError:
                 self._active = False
 
+        if self._engine.timing_logger:
+            end_time = datetime.datetime.now(SG_TIMEZONE.local)
+            duration = end_time - start_time
+            delay = start_time - event['created_at']
+            msg_format = "event_id=%d created_at=%s callback=%s start=%s end=%s duration=%s error=%s delay=%s"
```

Comment:
i know that this is coming from a different place in the code -- but my only nitpick, and this is my python ignorance, is that i can't figure out why the timestamp has a comma for the decimals place but the rest of the timing info uses a period.

Reply:
This is the logging Formatter using `asctime`. I probably should have used ISO but at this point, I don't want to break compatibility is people are parsing it to throw logs in some aggregation system.

The datetimes in the message use ISO format.

I've modified the output format of the durations to properly represent days if present in a more parseable format.